fossology/fossology
# feat(maintagent): add feature to delete orphaned records from database

## Description

Functionality to delete all the orphaned records from fossology database

### Changes

Removed Orphan records from uploadtree, clearing_event, clearing_decision, clearing_decision_event, obligation_map, obligation_candidate_maps table.

## How to test

* Upload a package Do clearing for all the files.
* Login to fossology database and delete the uploaded package from upload table.
* Login to fossology UI and add a candidate license and normal license.
* Add this licenses to obligations and delete the added licenses.
* Run maintagent and check if the all the orphaned records deleted from the above tables
